lib/pure/bitops.nim

```diff
@@ -27,6 +27,13 @@
 
 import macros
 import std/private/since
+from std/private/vmutils import forwardImpl, toUnsigned
+
+
+# sets.nim cannot import bitops, but bitops can use include
+# system/sets to eliminate code duplication. sets.nim defines
+# countBits32 and countBits64.
+import system/sets
```

Comment:
* now that sets is an import module, move it to `lib/std/private/builtinsets`

this way, we also avoid 2 import modules with same name:
lib/pure/collections/sets.nim
system/sets

which can cause issues (eg for docs https://nim-lang.github.io/Nim/sets.html, references etc)

* this comment also needs to be upgrade to reflect this and fact that it's import, not include

Reply:
For consistency with `channels_builtin` (#17330), perhaps use `sets_builtin`?

Reply:
sure, i updated the comment

Reply:
`countbits_impl` can also be moved to `std/private`, so I left it the following PR.